docs/configure.rst

```diff
@@ -0,0 +1,113 @@
+.. highlight:: console
+.. _configure:
+
+
+=========
+Configure
+=========
+
+
+Install Croud CLI
+=================
+
+This page shows you how to download and install the `Croud CLI`_. Croud is the dedicated command-line interface (CLI) tool to interact with CrateDB Cloud.
+The CLI provides specific commands to create and manage CrateDB Cloud
+organizations, projects, and resources. You can also use it to deploy CrateDB products such as clusters.
+
+In this tutorial, we refer to the *Croud CLI* as *Croud*.
+
+.. NOTE::
+
+   Croud requires `Python 3.6 or higher`_.
+
+.. SEEALSO::
```

Comment:
this should be moved to the end of the document

Reply:
doesn't the user need python before installing croud? if so, i think this info should be given upfront.

Reply:
I'm refering to the SEEALSO